boot/bootutil/src/loader.c

```diff
@@ -45,6 +45,10 @@
 #include "bootutil/security_cnt.h"
 #include "bootutil/boot_record.h"
 
+#ifdef CONFIG_SOC_NRF5340_CPUAPP
+#include "nrf53_cpunet_ctl.h"
```

Comment:
should we make it so that it requires you `pcd/nrf53_cpunet_ctl.h` so it would be easier to guess where this header is located?